packages/confirmation-modal/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -112,17 +122,19 @@ export const ConfirmationModal = React.forwardRef(
         </div>
         <Footer>
           <Button
+            {...confirmButtonProps}
+            data-testid={LGIDS_CONFIRMATION_MODAL.confirm}
+            disabled={!confirmEnabled || isConfirmDisabled}
+            className={cx(buttonStyle, confirmButtonProps?.className)}
             variant={variant}
-            disabled={!confirmEnabled || submitDisabled}
             onClick={handleConfirm}
-            className={buttonStyle}
-            data-testid={LGIDS_CONFIRMATION_MODAL.confirm}
           >
-            {buttonText}
+            {buttonText || confirmButtonProps?.children || 'Confirm'}
           </Button>
           <Button
+            {...cancelButtonProps}
             onClick={handleCancel}
-            className={buttonStyle}
+            className={cx(buttonStyle, cancelButtonProps?.className)}
             data-testid={LGIDS_CONFIRMATION_MODAL.cancel}
           >
             Cancel
```

Comment:
does cancel button text/children get overwritten? if it does, can we make this more explicit like with confirm button?

Reply:
I don't think this text is customizable, so I'll double-check with Rob.

Reply:
Confimed with Rob that this should always be `Cancel`

Reply:
It feels possible to me that a product team will want to customize this if they can customize the confirm text but maybe a case of YAGNI 😂

Reply:
Jeremy mentioned that he wanted to use this button as a "delete" action. It might be worth opening that up for customization as well